### ISSUE TITLE
Package version 1.7.0 doesn't work on Heroku

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-elixir#378]
I can run 1.6.7 fine, but deploying it with package 1.7.0 doesn't report any data.

```
# /tmp/appsignal/appsignal.log
[2018-07-12T11:17:13 (agent) #103][Warn] Connection error while posting to push api, queuing payload: Problem with the SSL CA cert (path? access rights?)
```

As reported in: https://app.intercom.io/a/apps/yzor8gyw/inbox/inbox/conversation/17438907239

**Comment by arjan** [comment]
@tombruijn having this error too. Whats the solution?

**Comment by tombruijn** [comment]
Hi @arjan, we fixed this in the patch release v1.7.1.
PR #380 was already merged, but the linked issue wasn't automatically closed because it was merged on master (instead of the "main" develop branch).

**Comment by arjan** [comment]
I still had the issue with 1.7.1 actually, but I already solved it, I did not install `ca-certificates` in my Alpine docker. Maybe good to mention somewhere.

**Comment by tombruijn** [comment]
With this change it should no longer be necessary to install the `ca-certificates` package since we bundle the CA certificate ourselves.

If you still have this issue on 1.7.1, do you move your app after compilation?
Does the ca_file_path location printed in the appsignal.log/app logs point to an existing location?

**Comment by arjan** [comment]
I dont see the path printed, dont have debugging enabled I think. However the appsignal log inside the container printed this:

```
[2018-07-16T09:53:27 (agent) #212][Info] Started appsignal-agent Config { local_env: {}, active: true, agent_standalone: false, running_in_container: true, app_path: Some("/app/lib/appsignal-1.7.1/priv"), agent_path: Some("/app/lib/appsignal-1.7.1/priv"), debug_logging: false, log: File, log_file_path: None, working_dir_path: "/tmp/appsignal", push_api_endpoint: "https://push.appsignal.com/2/collect", push_api_key: "b8dfc4e6-2ccf-4599-9e00-8455b76a8a2b", environment: "staging", app_name: Some("BotSquad"), language_integration_version: Some("elixir-1.7.1"), platform: None, transmission_interval: 30, http_proxy: None, ignore_actions: None, ignore_errors: None, ignore_namespaces: None, enable_host_metrics: true, hostname: Some("botsi-studio-deployment-5bbbd8984c-jttd8"), ca_file_path: None, diagnose_mode: false, dns_servers: None, files_world_accessible: true, app_revision: None, process_name: "process" }
[2018-07-16T09:53:27 (agent) #212][Info] Listening on /tmp/appsignal/agent.socket
[2018-07-16T09:53:57 (agent) #212][Warn] Connection error while posting to push api, queuing payload: Problem with the SSL CA cert (path? access rights?)
[2018-07-16T09:54:27 (agent) #212][Warn] Connection error while posting to push api, queuing payload: Problem with the SSL CA cert (path? access rights?)
```

Have not rebuilt the code since, only added the ca-certificates pkg; then it started working.

**Comment by arjan** [comment]
I think the default CA file path is not being written in the environment [here](https://github.com/appsignal/appsignal-elixir/blob/develop/lib/appsignal/config.ex#L212)

**Comment by tombruijn** [comment]
@arjan have you set `ca_file_path` to `nil` by any chance? Otherwise the option should be set with the default path to our bundled ca file.
If you set it to `nil` the `ca-certificates` package is required, otherwise not.

**Comment by arjan** [comment]
No I have not explicitly set anything..

**Comment by arjan** [comment]
But it is `nil` by default, because it is not part of the `@default_config` (it can't be, because the path needs to be determined in runtime). So I guess there should be a call to `default_ca_file_path` in `load_from_system()`?

**Comment by tombruijn** [comment]
@arjan right. We messed up with this config option again. I've submitted a fix in https://github.com/appsignal/appsignal-elixir/pull/381

**Comment by tombruijn** [comment]
@arjan the fix for this is released in package version 1.7.2.
